dss/storage/validator.py

```diff
@@ -44,6 +44,7 @@ def remove_json_fields(json_data: dict, path: List[str], fields: List[str]):
 )
 
 resolver = validators.RefResolver(referrer='', base_uri='')
+EXTRA_FIELD_ERROR_REGEX = re.compile(r"(?<!regexes: )'([^']+)',?")
```

Comment:
Nit: I would not extract the regex into a variable unless it is used in more than one place. Keep regexes as close to the place of use as possible. Especially the negative look behind makes more sense when the comment is right above it.

Reply:
Can you go over the json schema validator code to look for other types of error message that we haven't seen yet in the wild but that we might want to handle?

Reply:
My original reason for doing it was to avoid the cost of compiling the regex everytime `scrub_index_data` was called. The python `re` library caches the patterns so it actually isn't a problem. I'll move it back into the function.

Reply:
Good point. As long as it is right before the function it should be ok.